src/api/posts.js

```diff
@@ -1,17 +1,18 @@
 import axios from 'axios';
 import dayjs from 'dayjs';
 import config from '../constants/config';
+import convertData from '../utils/convertData';
 
 export default async (subreddit) => {
   try {
     const size = 500;
-    const date = dayjs()
-      .subtract(1, 'year')
-      .unix();
+    const date = dayjs().subtract(1, 'year').unix();
     const query = `?subreddit=${subreddit}&size=${size}&sort=desc&sort_type=score&after=${date}`;
     const URL = `${config.BASE_URL}search/submission/${query}`;
     const res = await axios.get(URL);
-    return res.data;
+
+    const convertedData = convertData(res.data.data);
```

Comment:
This is just a suggestion: Here you can directly return `convertData(res.data.data);`. I don't see a benefit in this constant.